Array_Rotation/README.md

```diff
@@ -0,0 +1,113 @@
+#Array Rotation
```

Comment:
`# Array Rotation` is the right way to use Markdown. Please have a look at preview.

Reply:
https://github.com/jainaman224/Algo_Ds_Notes/pull/2127/files?short_path=18564f4#diff-18564f4fa8ec225f109a0a6ba59f9efa